### PR TITLE
Add documentation about trouble shooting and MPI

### PR DESCRIPTION
**Pull request by InnocentBug**

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+To utilize full GPU with PySAGES CUDA support of HOOMD-blue, HOOMD-dlext and JAX are required.
+
+Once the installation requirements are satisfied, PySAGES can be installed with `pip`.::
+
+  
+   git clone https://github.com/SSAGESLabs/PySAGES.git
+   cd PySAGES
+   pip install .
+
+
```
Comment (pabloferz) [inline docs/source/installation.rst:25]:
```suggestion
   pip install git+https://github.com/SSAGESLabs/PySAGES.git

or

   git clone https://github.com/SSAGESLabs/PySAGES.git
   cd PySAGES
   pip install .
```

File: docs/source/installation.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+For the latest version of PySAGES, it is possible to deploy simulations with a `docker container <https://hub.docker.com/r/ssages/pysages>`__.
+At the moment we cannot guarantee a stable docker image for PySAGES.
+
+Our installation tutorial on Google Colab enables you to see how PySAGES,
```
Comment (pabloferz) [inline docs/source/installation.rst:29]:
```suggestion
Our installation tutorial on Google Colab shows how PySAGES,
```

File: docs/source/installation.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Once the installation requirements are satisfied, PySAGES can be installed with `pip`.::
+
+   pip install git+https://github.com/SSAGESLabs/PySAGES.git
+
+or
+
+   git clone https://github.com/SSAGESLabs/PySAGES.git
+   cd PySAGES
+   pip install .
```
Comment (pabloferz) [inline docs/source/installation.rst:26]:
```suggestion
    git clone https://github.com/SSAGESLabs/PySAGES.git
    cd PySAGES
    pip install .

```
I think I messed up the formatting with the previous suggestion

File: docs/source/trouble-shooting.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	  
+      Should be replaced by::
+	
+	 hoomd.context.initialize()
+	 context = hoomd.context.SimulationContext()
+	 with context:
+	   ...
+
+* My HOOMD-blue 2.x simulation crashes with a segfault after the simulation is finished.
+    * This is a known issue and we are trying to resolve it. For now, we hope that this does not interrupt your workflow.
```
[the remark this answers is not in the document]
Comment (InnocentBug) [reply docs/source/trouble-shooting.rst]: sure

File: docs/source/mpi.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
+Multiple GPU simulations via MPI
+================================
+
+PySAGES supports simulations with multiple GPUs for sampling methods that run multiple replicas of one simulation i.e. `pysages.methods.UmbrellaIntegration` and `pysages.methods.ImprovedString`.
```
Comment (pabloferz) [inline docs/source/mpi.rst:4]:
```suggestion
PySAGES supports simulations with multiple GPUs for sampling methods that run multiple replicas of one simulation i.e. `pysages.methods.UmbrellaIntegration` and `pysages.methods.SplineString`.
```

**Review by pabloferz — APPROVED** [review]

**Pull request merged**